src/report/ReportQueryRunner.ts

```diff
@@ -25,108 +23,119 @@ export enum QueryStatus {
  * @param setRecords  - callback to retrieve query records.
  * @param setFields - callback to set list of returned query fields.
  * @param queryTimeLimit - maximum query time in seconds.
- * @returns 
+ * @returns
  */
-export async function runCypherQuery(driver,
-    database = "",
-    query = "",
-    parameters = {},
-    rowLimit = 1000,
-    setStatus = (status) => { console.log("Query runner attempted to set status: " + JSON.stringify(status))},
-    setRecords = (records) => { console.log("Query runner attempted to set records: " + JSON.stringify(records))},
-    setFields = (fields) => { console.log("Query runner attempted to set fields: " + JSON.stringify(fields))},
-    fields = [],
-    useNodePropsAsFields = false,
-    useReturnValuesAsFields = false,
-    useHardRowLimit = false,
-    queryTimeLimit = 20,
+export function runCypherQuery(
+  driver,
+  database = '',
+  query = '',
+  parameters = {},
+  rowLimit = 1000,
+  setStatus = (status) => {
+    // eslint-disable-next-line no-console
+    console.log(`Query runner attempted to set status: ${JSON.stringify(status)}`);
```

Comment:
probably these console.log statements are not needed

Reply:
Not sure, if you confirm I can remove them but the functions setStatus, setRecords and setFields will be empty